### PR TITLE
Update changes

### PR DESCRIPTION
**Pull request by Josue-Dev-tech**
The GMS800L module is very magical and is used in many projects. It is in this ambition we provide this new library to help you in your different projects

### CODE REVIEW

**Comment by per1234** [comment]
Hello @Josue-Dev-tech. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

In the future, please take the time to clearly communicate the purpose of your pull request in the description. Even though some parts of this system are automated, other parts are still managed by humans and those humans will greatly appreciate that effort.

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/2096#issuecomment-1333518933

**Review by umbynos — APPROVED** [review]

**Pull request merged**
